### ISSUE TITLE
Shell is not working in MAUI Preview 14 for iOS

### ISSUE THREAD

**Issue by joker109** [dotnet/maui#5516]
### Description

MAUI apps with shell is not working in iOS. This is happening only after preview 14 update. The same app was working fine in earlier MAUI previews. 

Note: In android it's working fine.

![Screenshot 2022-03-23 at 6 29 13 PM](https://user-images.githubusercontent.com/34001302/159704772-3b8d12c8-ab1a-4921-9ea5-9bc46f1dd214.png)
[MAUIShellApp.zip](https://github.com/dotnet/maui/files/8333153/MAUIShellApp.zip)
.

### Steps to Reproduce

1. Create a new MAUI app.
2. Add a basic Shell structure.
3. Build and run in iOS.

### Version with bug

Preview 14 (current)

### Last version that worked well

Preview 13

### Affected platforms

iOS

### Affected platform versions

All Versions

### Did you find any workaround?

No

**Comment by jfversluis** [comment]
Duplicate of #5396 and already fixed. Should be good again in the next release!
